test/pages/print_labels/control_plates.spec.js

```diff
@@ -65,7 +65,7 @@ describe('print control plate labels', () => {
       data() {
         return {
           printer: 'heron-bc1',
-          numberOfBarcodes: 5,
+          numberOfBarcodes: '5',
```

Comment:
why a string?

Reply:
The input is coming in as a string.  As in, when the user changes the value in the form, the variable becomes a string.  This was overlooked because JS is so loosey goosey about types.  In fact I've noticed when you use a string containing an integer in a place where it wanted an actual integer, it mostly just treats it like an integer anyway.  I really don't like trusting that sort of behaviour and the language is a dog's dinner for supporting that sort of behaviour in the first place.